src/features/game/expansion/components/resources/Pebble.tsx

```diff
@@ -0,0 +1,254 @@
+
+  const [showPebbleTimeLeft, setShowPebbleTimeLeft] = useState(false);
+
+  const pebble = game.context.state.pebbles[pebbleIndex];
+  // Users will need to refresh to chop the tree again
+  const mined = !canMine(pebble);
+  const { setToast } = useContext(ToastContext);
+
+  // Reset the shake count when clicking outside of the component
+  useEffect(() => {
+    const handleClickOutside = (event: any) => {
+      if (
+        containerRef.current &&
+        !containerRef.current.contains(event.target)
+      ) {
+        setTouchCount(0);
+      }
+    };
+    document.addEventListener("click", handleClickOutside, true);
+    return () => {
+      document.removeEventListener("click", handleClickOutside, true);
+    };
+  }, []);
+
+  const displayPopover = async (element: JSX.Element) => {
+    setPopover(element);
+    setShowPopover(true);
+
+    await new Promise((resolve) => setTimeout(resolve, POPOVER_TIME_MS));
+    setShowPopover(false);
+  };
+
+  // Show/Hide Time left on hover
+
+  const handleMouseHoverPebble = () => {
+    if (mined) setShowPebbleTimeLeft(true);
+  };
+
+  const handleMouseLeavePebble = () => {
+    setShowPebbleTimeLeft(false);
+  };
+
+  const strike = () => {
+    const isPlaying = sparkGif.current?.getInfo("isPlaying");
+
+    if (!isPlaying) {
+      miningAudio.play();
+
+      sparkGif.current?.goToAndPlay(0);
+
+      setTouchCount((count) => count + 1);
+
+      // On second strike, mine
+      if (touchCount > 0 && touchCount === HITS - 1) {
+        mine();
+        miningFallAudio.play();
+        setTouchCount(0);
+      }
+    } else return;
+  };
+
+  const mine = async () => {
+    setTouchCount(0);
+
+    try {
+      gameService.send("pebble.struck", {
+        index: pebbleIndex,
+      });
+      setCollecting(true);
+      minedGif.current?.goToAndPlay(0);
+
+      displayPopover(
+        <div className="flex">
+          <img src={smallStone} className="w-5 h-5 mr-2" />
+          <span className="text-sm text-white text-shadow">{`+${pebble.stone.amount}`}</span>
+        </div>
+      );
+
+      setToast({
+        icon: smallStone,
+        content: `+${pebble.stone.amount}`,
+      });
+
+      await new Promise((res) => setTimeout(res, 2000));
+      setCollecting(false);
+    } catch (e: any) {
+      displayPopover(
+        <span className="text-xs text-white text-shadow">{e.message}</span>
+      );
+    }
+  };
+
+  const timeLeft = getTimeLeft(pebble.stone.minedAt, PEBBLE_RECOVERY_TIME);
+
+  return (
+    <div
+      className="relative z-10 w-full h-full"
+      onMouseEnter={handleMouseHoverPebble}
+      onMouseLeave={handleMouseLeavePebble}
+    >
+      {/* Unmined pebble which is strikeable */}
+      {!mined && (
+        <div
+          ref={containerRef}
+          className="group cursor-pointer w-full h-full"
+          onClick={strike}
+        >
+          <Spritesheet
+            className="group-hover:img-highlight pointer-events-none z-10"
```

Comment:
<img width="177" alt="Screen Shot 2022-07-08 at 9 27 42 am" src="https://user-images.githubusercontent.com/11745561/177887836-a3139afa-402c-4bce-8a01-2464e1cca69d.png">

Looking closely the pebble is a bit blurry.

Similar to the water PR, we need to make sure that a pixel width is the exact same throughout the entire game. In this case here the pebble pixel appears smaller than the grass pixels.

If you think a smaller pebble would look better, I would suggest designing a pebble that is smaller than the current one. Otherwise, scale it to ensure it fits the rest of the art in the game.

We need to be diligent with this and set the standard for the community. They will base their art of what we have in the game so it is extremely important we set the initial elements as pixel perfect 👍